src/overlays/actors/ovl_En_Dnt_Demo/z_en_dnt_demo.c

```diff
@@ -185,8 +185,7 @@ void EnDntDemo_Judge(EnDntDemo* this, PlayState* play) {
                 case PLAYER_MASK_ZORA:
                 case PLAYER_MASK_GERUDO:
                     rand9 = Rand_ZeroFloat(8.99f);
-                    maskIdx = Player_GetMask(play);
-                    maskIdx--;
+                    maskIdx = (s16)Player_GetMask(play) - 1;
```

Comment:
this looks fake to me (specifically the cast, I think it's more likely it was a short assignment like in main), can you add a `// fake` comment

Reply:
How would you feel about this?
```
// clang-format off
maskIdx = Player_GetMask(play); maskIdx--;
// clang-format on
```

Reply:
looks more plausible but formatting oddities suck so not sure if it's overall better
I guess it is slightly?

Reply:
either one can have a fake match comment for now and see if anyone can find something better during a fake match pass

Reply:
Alright, I left it as is and left the alternative in a comment